kernel/arch/i386/boot/bootparse.cpp

```diff
@@ -116,10 +122,179 @@ void px_parse_multiboot2(void *info)
             break;
         }
         default:
-            px_rs232_printf("Unknown Multiboot2 Tag: %d\n", tag->type);
+            px_rs232_printf("Unknown Multiboot2 tag: %d\n", tag->type);
             break;
         }
         // move to the next tag, aligning if necessary
         tag = (struct multiboot_tag*)(((uintptr_t)tag + tag->size + 7) & ~((uintptr_t)7));
     }
 }
+
+/*
+ *  ___ _   _          _     ___
+ * / __| |_(_)_ ____ _| |___|_  )
+ * \__ \  _| \ V / _` | / -_)/ /
+ * |___/\__|_|\_/\__,_|_\___/___|
+ */
+
+static const char* stivale2_mmap_type_to_string(uint32_t type)
+{
+    const char* str = "Invalid";
+    switch (type)
+    {
+        case STIVALE2_MMAP_USABLE:
+            str = "Available";
```

Comment:
Again super picky, but you can just return the string directly and that cuts down the number of lines by a factor of 2.

Reply:
Resolved in 8a32ed6